examples/rbac_policy.csv

```diff
@@ -2,4 +2,5 @@ p, alice, data1, read
 p, bob, data2, write
 p, data2_admin, data2, read
 p, data2_admin, data2, write
+p, alice, data, 1, 2, 3, read
```

Comment:
The policy rule ``p, alice, data, 1, 2, 3, read`` is a little weird. Because the format is supposed to be: ``p, sub, obj, act`` based on the following model:

https://github.com/casbin/gorm-adapter/blob/e74a050c51a4c12323e72f0c7ab445fb2daf6b40/examples/rbac_model.conf#L1-L14